src/spectrecoff/Spectre/Prompt.fs

```diff
@@ -44,26 +66,32 @@ module private Prompts =
 let private prompt prompter =
     AnsiConsole.Prompt prompter;
 
-let chooseFrom (choices: string list) question = 
+let chooseFrom (choices: string list) question =
     prompt (Prompts.selectionPrompt question choices)
 
-let chooseMultipleFromWith options (choices: string list) question = 
+let chooseMultipleFromWith options (choices: string list) question =
     prompt (Prompts.multiSelectionPrompt question choices options)
 
-let chooseMultipleFrom = 
+let chooseMultipleFrom =
     chooseMultipleFromWith defaultMultiSelectionOptions
 
-let ask<'T> question = 
+let chooseMultipleGroupedFromWith<'T> options (groupedChoices: ChoiceGroups<'T>) question =
```

Comment:
The only naming idea I have is to drop the _Multiple_ part, as you would not use groups if you only allow to select a single item. I'd then also switch the order of the words to `chooseFromGroupedWith`.

Reply:
You make a good point, done